code/game/objects/items/storage/bags.dm

```diff
@@ -77,6 +77,14 @@
 		to_chat(user, "<span class='warning'>You are unable to fit your [name] into the [J.name].</span>")
 		return
 
+/obj/item/storage/bag/trash/filled
+
+/obj/item/storage/bag/trash/filled/PopulateContents()
+	. = ..()
+	for(var/i in 1 to rand(0,6))

```

Comment:
By including zero here it means sometimes the trash will be empty, the loop won't run.
Is that intended? If not, set the minimum to `1`